web/packages/teleterm/src/ui/StatusBar/ShareFeedback/ShareFeedbackFormFields.tsx

```diff
@@ -76,19 +75,8 @@ export function ShareFeedbackFormFields({
       <FieldTextArea
         label="Suggestions"
         textAreaCss={`
-        font-size: 14px;
-        outline: none;
-        color: ${theme.colors.text.primary};
-        background: ${theme.colors.levels.surface};
-        border: 1px solid ${theme.colors.text.placeholder};
-        ::placeholder {
-          color: ${theme.colors.text.placeholder};
-        }
-        &:hover,
-        &:focus,
-        &:active {
-          border: 1px solid ${theme.colors.text.secondary};
-        }
+            background: inherit;
```

Comment:
This is to work around the fact that form elements are not aware of the surface level they're used at, right?

@rudream did you run into similar problems in the Web UI? This feels like an interesting problem to take care of in the future.

This is the background of the textarea that is shown when you click on the little chat icon in the bottom right corner.

Reply:
Well, maybe it's not even that interesting. If we want those field forms to be flat, they could just not set the background color by themselves and instead inherit it, as shown here.

Reply:
I think it makes sense to inherit parent color, but I was afraid to change it for `Input` which is used in a lot of places :/